### ISSUE TITLE
Bullseye Support?

### ISSUE THREAD

**Issue by skadz** [docker-library/redis#293]
Hi All -

  Debian Bullseye has been released and Buster has lots of security issues, any chance of a bullseye based image soon?

Thanks!
Ryan

**Comment by tianon** [comment]
> Debian Bullseye has been released and Buster has lots of security issues, ...

I agree that we should update to Bullseye, but because it is the latest stable release (definitely nothing to do with security) -- as a Debian Developer, I'm _very_ curious what security issues in Buster you're referring to?

Debian Buster is still actively supported by the Debian Security Team, and even next year when they no longer support it, it will transition to the LTS team, who will carry that torch for _another_ two years before it then transitions to the external "ELTS" team for even further support?
